### PR TITLE
wishlist

### PR DESCRIPTION
**Pull request by preethasri**
created wishlist feature 
livelink:https://deploy-preview-4--ornatestore.netlify.app/wishlist/wishlist.html

### CODE REVIEW

File: wishlist/wishlist.css [hunk 1]
```diff
@@ -0,0 +1,209 @@
...
+    --primary-btn: #6495ED;
+}
+
+
+/* nav bar */
+
+.landing-page-nav {
+    display: flex;
+    justify-content: space-between;
+    background-color: #f1f1f1;
```
Comment (someshtiwarii) [inline wishlist/wishlist.css:18]: try adding color variable so you can reuse it

**Review by someshtiwarii — APPROVED** [review]
rest looks good

File: wishlist/wishlist.html [hunk 2]
```diff
@@ -0,0 +1,182 @@
...
+    <meta charset="utf-8">
+    <meta name="viewport" content="width=device-width">
+    <title>wishlist</title>
+    <link href="wishlist.css" rel="stylesheet" type="text/css" />
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.css" integrity="sha512-5A8nwdMOWrSz20fDsjczgUidUBR8liPYU+WymTZP1lmY9G6Oc7HlZv156XqnsgNUzTyMefFTcsFH/tnJE/+xBg==" crossorigin="anonymous" referrerpolicy="no-referrer"
+    />
+    <link href="https://fonts.googleapis.com/css2?family=Fruktur&family=Inter:wght@200;400;600;700&family=Mulish&family=Nunito+Sans:wght@300&family=Poppins&family=Ubuntu:wght@300&family=Zen+Antique+Soft&display=swap" rel="stylesheet">
+  </head>
+  <body>
+    <div class="landing-page-nav">
```
Comment (suryxks) [inline wishlist/wishlist.html:13]: you can use nav instead of div here

**Review by suryxks — APPROVED** [review]
Rest looks good

File: wishlist/wishlist.html (hunk 2 above)
Comment (preethasri) [reply wishlist/wishlist.html:13]: sure.will check it out

File: wishlist/wishlist.css (hunk 1 above)
Comment (preethasri) [reply wishlist/wishlist.css:18]: thanks somesh will work on it

**Pull request merged**